src/axom/quest/tests/quest_initialize.cpp

```diff
@@ -6,6 +6,9 @@
 // Axom includes
 #include "axom/mint.hpp"
```

Comment:
We should probably explicitly include `axom/config.hpp` here 
(even though it is indirectly included by `axom/mint.hpp`)

Reply:
Done.